DiscordBot.Commands/SoundCommands/SoundRemoveCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿using DiscordBot.Commands.Core.CommandAttributes;
+using DiscordBot.Commands.Core;
+using Discord.WebSocket;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Commands.SoundCommands
+{
+    [Name("soundremove")]
+    [Description("Removes given sound")]
+    [Option("name", "Sound name to be removed", CommandOptionType.String)]
+    [RequiredPermission(Discord.GuildPermission.ManageGuild)]
+    public class SoundRemoveCommand : AbstractSoundCommand
+    {
+        private readonly ILogger<SoundRemoveCommand> _logger;
+
+        public SoundRemoveCommand(ILogger<SoundRemoveCommand> logger)
+        {
+            _logger = logger;
+        }
+
+        public override async Task ExecuteAsync(SocketSlashCommand command)
+        {
+            command.DeferAsync().Wait();
+
+            try
+            {
+                var name = command.GetRequiredOptionValue("name");
+
+                if(GetSoundNames().Contains(name))
```

Comment:
This basically duplicates the check from line 33 - `if(file.Exists)`. Lets remove this one and leave the one in line 33

Reply:
Replaced as suggested..